trunk/NDHMS/Routing/module_channel_routing.F

```diff
@@ -305,19 +339,44 @@ subroutine SUBMUSKINGCUNGE(    &
             if(D .eq. 0.0) then
               print *, "FATAL ERROR: D is 0 in MUSKINGCUNGE", Km, X, dt,D
               call hydro_stop("In MUSKINGCUNGE() - D is 0.")
-           endif
+           endif 
+ 
```

Comment:
Commented out lines from NWM v1.2. I copied the if statement on 355 later into the code to eliminate the possibility of negative flow due to excessive channel loss.